src/coresyftools/coresyftools/tests/test_tool.py

```diff
@@ -312,3 +312,42 @@ def test_can_use_custom_manifest_name(self):
         self.assertEqual(tool.manifest['name'], 'MyFairTool')
         os.remove(manifest_file_name)
 
+    def test_zip_output(self):
+        """Test the method to compress an output folder"""
+        tool = CoReSyFTool(self.runfile, 'manifest.json')
+        tool.write_to_zipfile('coresyftools/tests/tool6/myoutput/')
+        files_ = os.listdir('coresyftools/tests/tool6')
+        assert 'myoutput' in files_
+        archive = ZipFile('coresyftools/tests/tool6/myoutput', 'r')
+        self.assertGreater(len(archive.namelist()), 0)
+        archive.close()
+        os.remove('coresyftools/tests/tool6/myoutput')
+        os.rename('coresyftools/tests/tool6/myoutput_', 
+                    'coresyftools/tests/tool6/myoutput')
+
+    def test_zip_output_from_tool(self):
```

Comment:
I would also test (maybe in new functions) with multiple output options, and with options with lists of outputs. But I think this is also acceptable without such tests.